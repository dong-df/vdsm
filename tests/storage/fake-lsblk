#!/bin/sh
set -e

# Fake the output
cat $FAKE_STDOUT
