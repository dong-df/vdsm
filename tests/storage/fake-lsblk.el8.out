{
   "blockdevices": [
      {"name": "/dev/sda1", "type": "part",
         "children": [
            {"name": "/dev/sda", "type": "disk"}
         ]
      }
   ]
}
